README.md

```diff
@@ -52,7 +52,7 @@ To get latest devel code:
 Building multipath-tools
 ========================
 
-Prerequisites: development packages of for `libdevmapper`, `libreadline`,
+Prerequisites: development packages of for `libdevmapper`, `libedit`,
```

Comment:
This isn't quite right. We don't need either. The current default is to build without readline functionality, which can be optionally enabled with `READLINE=libedit` or `READLINE=libreadline`.

Reply:
Hmm, this is odd, I am pretty sure I had build fail because there was include of libedit, but I didn't have the -dev package. However now that I build it, it completes successfully, even though I don't currently have libedit. And indeed, I removed `readline-dev`, did a `git clean -fdx` and rebuilt the project — and it still completed without problems!

Strange, but okay, I'll fix the commit, sure

Reply:
> This isn't quite right. We don't need either.

I dived into the history, and it turned out that since the last I tested the code changed, and readline/editline was made optional. They were mandatory before.

Anyway, I presume I need to remove the libreadline from the deps list, and then add a text like:

> To enable functional X you might also pass `READLINE` variable set to `libedit` or `libreadline`, like `make READLINE=libreadline`, in which case you also need development package for the library you chose. Note that using libreadline may [make binary indistributable due to license incompatibility](https://github.com/opensvc/multipath-tools/issues/36).

My only question then, what is the functional `X` that the libs enable? Is there some interactive mode in `multipath` utility that readline enhances? That's usually what readline is for, but I don't see anything related in `multipath --help`

Reply:
> what is the functional X that the libs enable

It's the commandline history and TAB completion in the interactive mode which is entered with `multipathd -k` or `multipathc`. It works like command completion and history in `bash`. It's a feature that only few people are aware of, and the TAB completion has actually been non-functional until the current release (cc610cc, to be precise).

Reply:
I see, thanks! I reformulated the text, please see if you want anything changed, like the text, the style, or something